Snowdog_Components/components/Molecules/image/image.config.js

```diff
@@ -29,6 +29,14 @@ module.exports = {
         ],
         alt: 'image alt text'
       }
+    },
+    {
+      name: 'ratio',
+      context: {
+        wrapperClass: 'ratio-container',
+        class: 'ratio-image',
+        ratio: '100%'
```

Comment:
I think it would be good to include the whole calculation here `(imgHeight / imgWidth) * 100` so that it's clear how this works